core/src/main/java/org/stellar/anchor/auth/AuthConfig.java

```diff
@@ -8,7 +10,9 @@
 @NoArgsConstructor
 public class AuthConfig {
   AuthType type;
-  String secret;
+  String secretString;
+  @Nullable // Only not-null when type is JWT
+  SecretKey secretJwt;
```

Comment:
This should map to the auth configuration. We should move this to a helper function to avoid confusion.